libs/blocks/aside/aside.css

```diff
@@ -93,23 +122,44 @@
   margin-bottom: 0;
 }
 
-.aside .foreground.container .text .action-area>a {
+.aside .foreground.container .text .action-area > a {
   margin-right: var(--spacing-s);
 }
 
+.aside .foreground.container .text .action-area > a:last-child {
+  margin-right: 0;
+}
+
+.aside .foreground.container .text .action-area > a:not(.con-button) {
+  text-decoration: underline;
+}
+
 .aside .foreground.container div {
   flex: 1 1 auto;
 }
 
-.aside .foreground.container .image {
+.aside .foreground.container .image,
+.aside .foreground.container div:has(.milo-video),
+.aside .foreground.container .milo-inline-video {
+  display: flex;
   margin-top: 32px;
```

Comment:
This seems weird to have set pixel value on top and variable on bottom. Id use vars for both if possible

Reply:
fixed